### PR TITLE
Feature/istio endpoint scan

### LINKED ISSUE #437: Create a brew tap for mesheryctl through goreleaser
**Current Behavior**
We want `mesheryctl` to be installable through homebrew. We already have `goreleaser` as part of our CI with GitHub actions. `goreleaser` has a way to publish brew formula for every release described here: `https://goreleaser.com/homebrew/`.  Following the steps here we want to update our `goreleaser` configuration to publish a new version of the brew formula.
We will want to first create a new repo to publish the formula.

**Expected Behavior**
- As part of every release we also want to publish a new brew formula.
- brew users can add this repo as a tap and then install `mesheryctl`.

### PR DESCRIPTION
**Pull request by leecalcote**
- goreleaser for homebrew tap
- nsm adapter in UI
- switch from badger to bitcask
- initial logic for prometheus and Grafana discovery.

fixes #437

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,7 +50,7 @@ run-local:
 	SAAS_BASE_URL="https://meshery.layer5.io" \
 	PORT=9081 \
 	DEBUG=true \
-	ADAPTER_URLS="mesherylocal.layer5.io:10000 mesherylocal.layer5.io:10001" \
+	ADAPTER_URLS="mesherylocal.layer5.io:10000 mesherylocal.layer5.io:10001 mesherylocal.layer5.io:10004" \
```
Comment (leecalcote) [inline Makefile]: @girishranganathan should we bring in Octarine here as well?

Comment (girishranganathan) [reply Makefile]: yes it was already done in the master. Now I have syncd with master and it should be all fine.

Comment (leecalcote) [reply Makefile]: Thank you. Good to merge?

**Comment by leecalcote** [comment]
Builds and runs locally for me.

**Pull request merged**